mu_repo/__init__.py

```diff
@@ -69,6 +69,28 @@ def CreateConfig(config_file='.mu_repo'):
 def CreateParams(args, config_file='.mu_repo'):
     return Params(CreateConfig(config_file), args, config_file)
 
+
+def SearchConfigFile(start_dir, recurse_limit=20):
+    """Search upwards in the file system starting in start_dir for a ".mu_repo" file, until the maximum limit or
+    the root of the file-system is reached.
+    """
+    start_dir = os.path.normpath(os.path.abspath(start_dir))
```

Comment:
Humm, I think removing the normpath (but keeping the abspath) is safe as abspath should do a normpath internally anyways (based on the comments from https://docs.python.org/2/library/os.path.html, abspath should always return normalized).